src/openrct2/ride/rtd/coaster/CorkscrewRollerCoaster.h

```diff
@@ -79,7 +79,7 @@ constexpr RideTypeDescriptor CorkscrewRollerCoasterRTD =
             { RatingsModifierType::BonusScenery,          0,                6693, 0, 0 },
             { RatingsModifierType::RequirementDropHeight, 12,               2, 2, 2 },
             { RatingsModifierType::RequirementMaxSpeed,   0xA0000,          2, 2, 2 },
-            { RatingsModifierType::RequirementNegativeGs, FIXED_2DP(0, 40), 2, 2, 2 },
+            { RatingsModifierType::RequirementNegativeGs, MakeFixed2dp<fixed16_2dp>(0, 40), 2, 2, 2 },
```

Comment:
I think it would be worth adding a `makeFixed16_2dp`, considering the verbosity of the current syntax in these tables.